java/quickstart/pom.xml

```diff
@@ -43,7 +43,7 @@ limitations under the License.
     <dependency>
       <groupId>com.google.cloud.bigtable</groupId>
       <artifactId>bigtable-hbase-1.x</artifactId>
-      <version>1.4.0</version>
+      <version>1.9.0</version>
```

Comment:
Why not `2.x` then? Just curious.

Reply:
We support both `hbase-1.x` & `2.x` version and would continue it for at least next few quarters, So I tried not to change the original intent for these examples.

Reply:
We should continue with 1.x for now.  Most of our users use 1.x.